helix-term/src/keymap/default.rs

```diff
@@ -38,6 +38,7 @@ pub fn default() -> HashMap<Mode, KeyTrie> {
         "G" => goto_line,
         "g" => { "Goto"
             "g" => goto_file_start,
+            "|" => goto_column,
```

Comment:
This should also be added to the book https://github.com/helix-editor/helix/blob/master/book/src/keymap.md#goto-mode